### PR TITLE
- Added WindowConfirm component

### PR DESCRIPTION
**Pull request by minotogna**
- Using WindowConfirm to render survey publish confirm message

### CODE REVIEW

File: core/i18n/resources/en.js [hunk 1]
```diff
@@ -145,7 +146,18 @@ export const enTranslation = {
...
+      
+Please, note that:
+
+- All labels associated with deleted languages will be permanently deleted.
+- Node definitions cannot be changed from single to multiple and vice versa.
+- All category items cannot change code and cannot be deleted.
+- All taxa cannot be modified.
+- All records associated with deleted cycles will be permanently deleted.
+- All nodes associated with deleted node definitions will be permanently deleted.
+
```
Comment (eliask) [inline core/i18n/resources/en.js]: Maybe rephrase as follows?

```
- Category items' codes cannot be changed after publishing
- Category items cannot be deleted after publishing
- Existing taxa cannot be modified after publishing
```

Although I'm not sure about the taxa. Can new taxa be added? Can they still be deleted?

Also, is "node" something that is obvious to the end user? I haven't seen "node" mentioned anywhere in the UI except for error messages.

Comment (fefrain) [reply core/i18n/resources/en.js]: I agree with Elias, you should specify "after publishing" somewhere in the message, maybe in the beginning of it: Please not that after publishing: ...

File: webapp/commonComponents/windowConfirm.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
+import './windowConfirm.scss'
```
Comment (fefrain) [inline webapp/commonComponents/windowConfirm.js]: I would call this module ConfirmDialog...

File: core/i18n/resources/en.js (hunk 1 above)
Comment (minotogna) [reply core/i18n/resources/en.js]: how about this

```#### You are about to publish the survey {{survey}} ####

###### During publish: ###### 
- All labels associated with deleted languages will be permanently deleted.
- All records associated with deleted cycles will be permanently deleted.
- Data associated with deleted form fields will be permanently deleted.

###### After publish: ###### 
- Form fields cannot be changed from single to multiple and vice versa.
- Category items' codes cannot be changed.
- Category items cannot be deleted.
- Taxa cannot be modified.

** Are you sure you want to continue? **```

File: webapp/commonComponents/windowConfirm.js (hunk 2 above)
Comment (minotogna) [reply webapp/commonComponents/windowConfirm.js]: ok

File: core/i18n/resources/en.js (hunk 1 above)
Comment (minotogna) [reply core/i18n/resources/en.js]: i changed the message
